### PR TITLE
chore: onboard adobe to proxy v1 tests

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## What are the changes introduced in this PR?

Onboard adobe analytics to proxy v1 tests
Any new scenarios added = n/a

## What is the related Linear task?

Resolves INT-1477

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3163/test-report.html

**Comment by utsabc** [comment]
Can we add `other.ts` scenarios tests as well?

**Comment by yashasvibajpai** [comment]
> Can we add `other.ts` scenarios tests as well?

We do not have possibility for other scenarios for Adobe, the scenarios are covered in business

File: test/integrations/destinations/adobe_analytics/dataDelivery/business.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+const headers = {
+  'Content-Type': 'application/xml',
+};
+
+export const reqMetadataArray = [proxyMetdata];
+
+const failureRequestParameters = {
+  XML: {
+    payload:
+      '<?xml version="1.0" encoding="utf-8"?><request><browserHeight>1794</browserHeight><browserWidth>1080</browserWidth><campaign>sales campaign</campaign><channel>web</channel><currencyCode>USD</currencyCode><ipaddress>127.0.0.1</ipaddress><language>en-US</language><userAgent>Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)</userAgent><referrer>https://www.google.com/search?q=estore+bestseller</referrer><marketingcloudorgid/><events>prodView</events><products>Games;;11;148.39</products><reportSuiteID>failureReport</reportSuiteID></request>',
```
Comment (koladilip) [inline test/integrations/destinations/adobe_analytics/dataDelivery/business.ts:37]: this repeated

Comment (koladilip) [reply test/integrations/destinations/adobe_analytics/dataDelivery/business.ts:37]: image

**Review by utsabc — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

File: test/integrations/destinations/adobe_analytics/dataDelivery/business.ts (hunk 1 above)
Comment (yashasvibajpai) [reply test/integrations/destinations/adobe_analytics/dataDelivery/business.ts:37]: WIll handle in the upcoming network handler card

**Review by shrouti1507 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**